tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2958,6 +2958,26 @@ public function userListsTheActivitiesForFolderOfSpaceWithDepthOrLimitUsingTheGr
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @Then the activities should be in the following order:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theActivitiesShouldBeInTheFollowingOrder(TableNode $table): void {
+		$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent();
+		$activities = $responseBody->value;
+
+		foreach ($table->getHash() as $index => $expectedValue) {
+			$actualActivity = $activities[$index];
+			$expectedActivity = $expectedValue['resource'] . ":" . $expectedValue['message'];
+			$actualActivity = $actualActivity->template->variables->resource->name . ":" . $actualActivity->template->message;
+			Assert::assertEquals($expectedActivity, $actualActivity, "Expected: '$expectedActivity' but got '$actualActivity'");
```

Comment:
we can just ho with this short error message. because the assertion will show what was expected and what found.
I promise this will be the last comment. :sweat_smile: 
```suggestion
			Assert::assertEquals($expectedActivity, $actualActivity, "Activity didn't match");
```

Reply:
> we can just ho with this short error message. because the assertion will show what was expected and what found. I promise this will be the last comment. 😅

Yeah i was also confused on this one as well.


The last comment....until the next one! 😆